package.json

```diff
@@ -48,7 +48,7 @@
     "karma-phantomjs-launcher": "^0.2.0",
     "karma-phantomjs-shim": "^1.0.0",
     "karma-sinon-chai": "^1.0.0",
-    "karma-webpack": "^1.5.1",
+    "karma-webpack": "1.5.1",
```

Comment:
sounds like updating karma and karma-webpack would fix it too? https://github.com/webpack/karma-webpack/issues/65#issuecomment-126675635